manifests/candlepin.pp

```diff
@@ -49,6 +70,13 @@
 
   if $deploy {
 
+    Cert[$tomcat_cert_name] ~>
```

Comment:
I agree. Let's make a list of issues with the certs module. I have a few more items to share. :-)